pkg/didcomm/protocol/didexchange/models.go

```diff
@@ -101,3 +91,126 @@ type Connection struct {
 	DID    string   `json:"did,omitempty"`
 	DIDDoc *did.Doc `json:"did_doc,omitempty"`
 }
+
+// ReceiveInvitationRequest model
+//
+// This model defines DID exchange receive invitation request
+//
+type ReceiveInvitationRequest struct {
+
+	// the Type of the connection invitation
+	Type string `json:"@type,omitempty"`
+
+	// the ID of the connection invitation
+	ID string `json:"@id"`
+
+	// the Service endpoint of the connection invitation
+	ServiceEndpoint string `json:"serviceEndpoint,omitempty"`
+
+	// the Label of the connection invitation
+	Label string `json:"label,omitempty"`
+
+	// the RecipientKeys for the connection invitation
+	RecipientKeys []string `json:"recipientKeys,omitempty"`
+}
+
+// ReceiveInvitationResponse model
+//
+// This model defines DID exchange receive invitation response
+//
+type ReceiveInvitationResponse struct {
+
+	//State of the connection invitation
+	State string `json:"state"`
+
+	//Created time
+	CreateTime time.Time `json:"created_at,omitempty"`
+
+	//Updated time
+	UpdateTime time.Time `json:"updated_at,omitempty"`
+
+	// the connection ID of the connection invitation
+	ConnectionID string `json:"connection_id"`
+
+	//Routing state of connection invitation
+	RoutingState string `json:"routing_state,omitempty"`
+
+	//Connection invitation initiator
+	Initiator string `json:"initiator,omitempty"`
+
+	//Connection invitation accept mode
+	Accept string `json:"accept,omitempty"`
+
+	//Invitation mode
+	Mode string `json:"invitation_mode,omitempty"`
+
+	//Request ID of invitation response
+	RequestID string `json:"request_id"`
+
+	//My DID
+	DID string `json:"my_did"`
+
+	//Invitation key
+	InvitationKey string `json:"invitation_key,omitempty"`
+
+	//Other party's label
+	InviterLabel string `json:"their_label,omitempty"`
+}
+
+// AcceptInvitationResponse model
+//
+// This model defines DID exchange accept invitation response
+//
+type AcceptInvitationResponse struct {
```

Comment:
Same here - this doesn't belong in a didcomm protocol package

Reply:
Same response as above, this is not part of REST API request/response model.
Because of same name it is causing confusion.

Reply:
removed